c2corg_ui/static/partials/cards/outings.html

```diff
@@ -10,10 +10,13 @@
     <span class="list-item-lang" ng-if="cardCtrl.lang != locale.lang">({{::locale.lang}})</span>
     <br>
     <div class="quality" ng-if="::doc['quality']" ng-class="::doc['quality']"
-         uib-tooltip="{{'quality'| translate}} : {{::doc['quality'] | translate}}" tooltip-placement="left"></div>
+         uib-tooltip="{{'quality'| translate}} : {{::doc['quality'] | translate}}" tooltip-placement="left">
+    </div>
 
-    <span class="elevation-max" ng-if="::doc['elevation_max']" uib-tooltip="{{'elevation_max' | translate}}">
-      <span translate class="value-title glyphicon glyphicon-sort-by-attributes"></span>
+    <!--elevation-max is shown only in the home feed-->
+    <span class="elevation-max" ng-if="::doc['elevation_max'] && cardCtrl.currentLocation == '/'"
```

Comment:
> So I could test for "/" and "/profiles/".

Again, this is not robust. What if the feed is integrated somewhere else? We won't add all the possible URLs in the test :) A robust way would be that the cards know (via a directive attribute?) that they are used in a feed.

Anyway I think we should simply have this ``elevation_max`` info in all outings cards, feed or not.
By the way it seems this attribute is already available in the cards of the advanced search!?

So why don't you drop this condition and reuse what is done for the advanced search cards?